www/js/pages/maltainvest-account-opening/maltainvest-account-opening.template.html

```diff
@@ -457,15 +457,17 @@ <h1 class="page-title">
           <div class="row">
             <div class="col input-title">
               {{ 'new-account-maltainvest.address_postcode' | translate }}
-              <span>{{ 'new-account-maltainvest.is_optional' | translate }}</span>
+              <span ng-if="!vm.hasIOM">{{ 'new-account-maltainvest.is_optional' | translate }}</span>
             </div>
           </div>
           <div class="row">
             <div class="col item-input">
               <input name="address_postcode"
                      ng-model="vm.data.address_postcode"
                      ng-pattern="vm.validation.validatePostcode"
-                     ng-maxlength="vm.validation.length.postcode.max"/>
+                     ng-maxlength="vm.validation.length.postcode.max"
+                     ng-required="vm.hasIOM"
```

Comment:
@nazaninreihani `vm.hasIOM` is a function but you use it as a variable. so you should call it `vm.hasIOM()`.